providers/google/internal/mail/README.md

```diff
@@ -0,0 +1,17 @@
+# Read
+
+Objects suitable for reading from Gmail APIs:
+* [Drafts](https://developers.google.com/workspace/gmail/api/reference/rest/v1/users.drafts/list) - paginated
+* [Filters](https://developers.google.com/workspace/gmail/api/reference/rest/v1/users.settings.filters/list)
+* [Labels](https://developers.google.com/workspace/gmail/api/reference/rest/v1/users.labels/list)
+* [Messages](https://developers.google.com/workspace/gmail/api/reference/rest/v1/users.messages/list) - paginated
+* [SendAs](https://developers.google.com/workspace/gmail/api/reference/rest/v1/users.settings.sendAs/list)
```

Comment:
Instead of having `sendas` & `filters` as objects, are we able to define `settings` as an object that gets all settings?